client/controllers/curatorInbox/addToEvent.coffee

```diff
@@ -24,15 +25,19 @@ Template.addToEvent.onRendered ->
       placeholder: 'Search for an Event...'
       minimumInputLength: 0
 
-    $(document).on 'click', '.add-new-event', (event) ->
+    $(document).on 'click', ".add-new-event-#{instanceData.objNameToAssociate}", (event) ->
```

Comment:
Do you know why a document click handler is being used here instead of a template click handler? I think this might cause bugs since nothing appears to clean up the old handlers.